### ISSUE TITLE
Path in warning message is badly formatted

### ISSUE THREAD

**Issue by eschnett** [LLNL/spack#1148]
I just received this warning from Spack:
```
  Spack will think this package is installed. Manually remove this
  directory to fix:
  /ddnA/project/eschnett/shelob/src/spack/opt/spack/linux-
  redhat6-x86_64/gcc-6.1.0-spack/julia-master-
  65fs5pr57mv6jfcj7cqvu3ipdw4n56uo
```
The path name should not be broken over multiple lines.

**Comment by citibeth** [comment]
I fixed this in #543.  I should probably go away when that is merged.

**Comment by tgamblin** [comment]
Fixed by #543.
